#5 Controller, service, Company and CompanyDto classes added

svgogin opened the pull request:
Add spring-boot-web dependency (done)
Add simple dto class for your controller (partially done, had some problems with conversion Dto class to Entity class)
Add controller with your dto class and 2 methods (done)

Commit:
#5 Controller, service, Company and CompanyDto classes added
Add spring-boot-web dependency (done)
Add simple dto class for your controller (partially done, had some problems with conversion Dto class to Entity class)
Add controller with your dto class and 2 methods (done)

alinkov commented on pom.xml:
```diff
@@ -12,7 +12,7 @@
 	<version>0.0.1-SNAPSHOT</version>
 	<name>spark</name>
-	<description>Spark service for study purposes</description>
+	<description>Spark ru.svgogin.service.spark.service for study purposes</description>
```
restore plz
```suggestion
	<description>Spark service for study purposes</description>
```

alinkov commented on src/main/java/ru/svgogin/service/spark/dto/CompanyDto.java:
```diff
@@ -0,0 +1,52 @@
+  private final String inn;
+  private final String ogrn;
+  private final String kpp;
+  private final String full_name_rus;
```
Use camelCase in java.
```suggestion
  private final String fullNameRus;
```
You can find some guides here:
- https://google.github.io/styleguide/javaguide.html
- http://cr.openjdk.java.net/~alundblad/styleguide/index-v6.html
- https://www.oracle.com/java/technologies/javase/codeconventions-contents.html

alinkov commented on src/main/java/ru/svgogin/service/spark/entity/Company.java:
```diff
@@ -0,0 +1,52 @@
+  private String inn;
+  private String ogrn;
+  private String kpp;
+  private String full_name_rus;
```
also rename properties

alinkov commented on src/main/java/ru/svgogin/service/spark/entity/Company.java:
```diff
@@ -0,0 +1,52 @@
+  private String full_name_rus;
+  private String short_name_rus;
+  private String status_name;
+  private Date status_date;
```
don't use java.util.Date.
Use java 8 date classes, e.g. OffsetDateTime, LocalDateTime, LocalDate, etc.

alinkov commented on src/main/java/ru/svgogin/service/spark/service/SparkService.java:
```diff
@@ -0,0 +1,48 @@
+    save(new Company("7725038124","1037739527077","770401001","АКЦИОНЕРНОЕ ОБЩЕСТВО \"БАНК ДОМ.РФ\"","АО\"БАНК ДОМ.РФ\"","Действующая",
+        new Date()));
+  }
+  public Iterable<Company> findAll() {
```
add new lines between methods.

Commit:
#5 Controller, service, Company and CompanyDto classes added
Add spring-boot-web dependency (done)
Add simple dto class for your controller (partially done, had some problems with conversion Dto class to Entity class)
Add controller with your dto class and 2 methods (done)

Commit:
#5 Controller, service, Company and CompanyDto classes added
Add spring-boot-web dependency (done)
Add simple dto class for your controller (partially done, had some problems with conversion Dto class to Entity class)
Add controller with your dto class and 2 methods (done)

alinkov commented on pom.xml:
```diff
@@ -12,7 +12,7 @@
 	<version>0.0.1-SNAPSHOT</version>
 	<name>spark</name>
-	<description>Spark service for study purposes</description>
+	<description>ru.svgogin.service.spark</description>
```
```suggestion
	<description>Spark service for study purposes</description>
```

Commit:
#5 Controller, service, Company and CompanyDto classes added
Add spring-boot-web dependency (done)
Add simple dto class for your controller (partially done, had some problems with conversion Dto class to Entity class)
Add controller with your dto class and 2 methods (done)
Type date was changed to LocalDate.
Description was changed to Spark service for study purposes
